app/assets/stylesheets/dashboard/service_calendar.sass

```diff
@@ -1,3 +1,14 @@
+.user-view-ssr-modal
+  width: 95%
+
+.visit-before-container, .visit-day-container, .visit-after-container
```

Comment:
If I understand Kyle correctly, I'm with him on this one. As far as I can tell, you are removing bootstrap classes from the views, and replacing them with your own custom classes, only to then import the styling mixin in the css, and assign it to your new custom selector. This will make for oversized custom css files eventually, and make maintaining styles across the site a nightmare. One of the benefits of using bootstrap in the first place is that you have a universal set of classes to use in your markup, and can avoid constantly creating your own (which then almost always leads to duplication, and unintended styling bugs).